sdk/healthinsights/azure-health-insights-radiologyinsights/README.md

```diff
@@ -67,9 +72,26 @@ Build an **asynchronous** client:
 String endpoint = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_ENDPOINT");
 String apiKey = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_API_KEY");
 
-RadiologyInsightsAsyncClient radiologyInsightsAsyncClient = new RadiologyInsightsClientBuilder()
-        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest())
-        .credential(new AzureKeyCredential(apiKey)).buildAsyncClient();
+RadiologyInsightsClientBuilder clientBuilder = new RadiologyInsightsClientBuilder()
+        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest());
+if (apiKey != null && !apiKey.equals("")) {
+    clientBuilder = clientBuilder.credential(new AzureKeyCredential(apiKey));
+}
+RadiologyInsightsAsyncClient radiologyInsightsAsyncClient = clientBuilder.buildAsyncClient();
+```
+
+#### Authenticate and create a RadiologyInsightsClient with DefaultAzureCredential
+
+DefaultAzureCredential provides an alternative way to authenticate with the service. Documentation about this can be found [here][azure_credential]
+
+```java com.azure.health.insights.radiologyinsights.defaultazurecredential
+String endpoint = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_ENDPOINT");
+
+RadiologyInsightsClientBuilder clientBuilder = new RadiologyInsightsClientBuilder()
+        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest());
+DefaultAzureCredential c = new DefaultAzureCredentialBuilder().build();
+
+clientBuilder = clientBuilder.credential(c);
```

Comment:
Why not code them together? Also one usually does not need to set `serviceVersion`. e.g.

```
RadiologyInsightsClientBuilder clientBuilder = new RadiologyInsightsClientBuilder()
        .endpoint(endpoint)
        .credential(credential);
RadiologyInsightsAsyncClient radiologyInsightsAsyncClient = clientBuilder.buildAsyncClient();
```

Reply:
Changed as suggested, the code is indeed cleaner like that!